src/StudyEntities/Candidate/ValidatableIdentifier.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace LORIS\StudyEntities\Candidate;
+
+abstract class ValidatableIdentifier implements Identifier
+{
+    protected $value;
+
+    abstract static protected function validate(string $value): bool;
+
+    public function __construct(string $value)
+    {
+        if (!static::validate($value)) {
```

Comment:
`\static` instead of `static`?

Reply:
I could be mistaken, but I don't think language keywords are affected by namespacing.